api/server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express')
+const games = require('../models/gamesModel')
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/games', async (req, res) => {
+  const allGames = await games.getAllGames();
+
+  res.status(200).json(allGames)
+})
+
+server.get('/games/:id', async (req, res) => {
+  const { id } = req.params;
+
+  const game = await games.getGameById(id)
+  res.status(200).json(game)
+})
+
+server.post('/games', async (req, res) => {
+  const body = req.body;
+
+  if(body.title && body.genre) {
+    try {
+      await games.insert(body)
```

Comment:
If you made those other changes
```suggestion
      await games.insert(req.body)
```